src/content/docs/en/welcome.mdx

```diff
@@ -124,21 +124,21 @@ However, Ultramarine Linux includes Terra, RPM Fusion and Flathub.
 ### Why Ultramarine (instead of Windows)
 
 - Performance
-  - Windows bloats your computer with many different potentially unwanted programs that slow down your computer. Ultramarine does not.
-  - Ultramarine provides a few lightweight desktop environments (namely Budgie/Flagship and XFCE), which run faster and provide a snappier experience.
-  - Ultramarine uses newer technologies that are not available on Windows because of backwards compatibility. These technologies brings better performance.
+  - Windows ships many different potentially unwanted programs that slow down your computer. Ultramarine does not.
+  - Ultramarine provides a few lightweight desktop environments (namely Budgie/Flagship and XFCE) that offer more control to the average user without harming performance and usability.
+  - Ultramarine uses newer technologies that are not available on Windows because of backwards compatibility. These technologies bring better performance.
     - For example, task schedulers, file systems, shells, etc.
     - Most modern servers (including giant web servers) use Linux simply because it's much faster.
 - Privacy
-  - We do not collect/track your private data ever because that's a waste of storage.
-  - We don't even know how many Ultramarine devices there are (unlike some companies).
-  - When you setup Windows, you need to accept a very long license. Well, we don't even have a license agreement page in our installer.
-  - Ultramarine is open source. Anyone can check if we track/collect your data by reviewing the source code. We don't.
+  - We do not collect/track personally identifiable information. We have no use for it.
+  - We have no way of tracking how many people use Ultramarine.
+  - When you setup Windows, you need to accept a EULA (end-user license agreement) that is intentionally difficult to follow. Ultramarine uses free software licenses while providing the user with the option to use proprietary software for the sake of usability.
+  - Ultramarine is open source. Anyone can check if we track/collect your data by reviewing the [source code](https://github.com/Ultramarine-Linux).
 - Security
-  - Security updates are much quicker than Windows.
-    - Updates don't require reboots.
-  - It's much more unlikely for malware, viruses, etc. to target Linux due to its current low popularity.
-  - Our open source model makes it very easy for anyone to spot bugs/vulnerabilities and fix them quicker.
+  - Security updates are more frequent and stable than Windows.
+    - Most userland updates don't require reboots.

```

Comment:
add an asterisk, they don't *have to* require reboots but most editions do offline updates for stability reasons. this is still faster than Windows despite needing a reboot

you can still get around this by just updating the system with DNF